Jenkinsfile

```diff
@@ -74,8 +74,8 @@ node('mesos-ubuntu') {
   }
 
   stage("Get packer") {
-    shcmd("""apt-get install -y curl &&
-          curl -L -O https://releases.hashicorp.com/packer/1.2.4/packer_1.2.4_linux_amd64.zip &&
+    shcmd("""apt-get install -y wget &&
```

Comment:
btw, this is fine!. but most often curl is more commonly available than wget. The retries mechanism in curl are better and more feature rich tha wget.

That's why in dcos we use curl a lot.